packages/netlify-plugin-cache-docs/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "netlify-plugin-cache-docs",
-  "version": "5.0.0-alpha.25",
+  "version": "5.0.0-alpha.26",
```

```suggestion
  "version": "5.0.0-alpha.34",
```

@oliviertassinari should we change this too? I know it's private but it would avoid confusion in the future.

---

Did the package change?

---

it changed from `25` to `26`, @oliviertassinari suggest to update to `34` for consistency?

---

I'm at a loss. Why was my comment ignored?

---

My mistake, I should have waited for olivier response

---

No preferences. Since this package is private, we could have either leave the version unchanged (26) or use the latest version (34).